src/pages/Exchange/Exchange.js

```diff
@@ -801,9 +816,7 @@ export const Exchange = forwardRef((props, ref) => {
         type="button"
         onClick={onMultipleCancelClick}
       >
-        <Trans>
-          Cancel {cancelOrderIdList.length} {orderText}
-        </Trans>
+        <Plural value={cancelOrderIdList.length} one="Cancel # order" other="Cancel # orders" />
```

Comment:
for one it will be "Cancel 1 order" or "Cancel order"? I guess the last is preferable

Reply:
Fixed!